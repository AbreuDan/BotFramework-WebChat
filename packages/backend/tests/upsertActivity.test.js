import createFacility from './createFacility';

test('Upsert activity', async () => {
  const { directLine, store } = await createFacility();

  directLine.activity.next({
    text: 'Hello, World!',
    type: 'message'
  });

  await 0;

  expect(store.getState().activities).toMatchSnapshot();
});
