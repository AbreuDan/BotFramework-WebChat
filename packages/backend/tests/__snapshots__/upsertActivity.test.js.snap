// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Upsert activity 1`] = `
Array [
  Object {
    "text": "Hello, World!",
    "type": "message",
  },
]
`;
